src/main/java/repository/BookRepository.java

```diff
@@ -0,0 +1,9 @@
+package repository;
+
+import model.Book;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.stereotype.Repository;
+
+@Repository
+public interface BookRepository extends JpaRepository<Book, Long> {
```

Comment:
you should implement methods 
```
Book save(Book book);
List<Book> findAll();
```
```suggestion
public interface BookRepository {
```